.github/workflows/directoriesFilesChangePR.yml

```diff
@@ -0,0 +1,129 @@
+name: 'Directories/Files Change Test PR'
+on:
+  pull_request:
+    types: [ready_for_review]
+    paths-ignore:
+      - '.gitignore'
+      - '*.md'
+      - 'doc/**'
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.github/*.yml'
+      - 'buildenv/**'
+  push:
```

Comment:
I would say pull_request trigger is enough, which could save some compute cycles against our GHA machine time allocation (if such a thing exists).

Reply:
Fixed (Removed the push trigger).